espnet2/tasks/asr.py

```diff
@@ -429,9 +439,11 @@ def optional_data_names(
         cls, train: bool = True, inference: bool = False
     ) -> Tuple[str, ...]:
         MAX_REFERENCE_NUM = 4
-        retval = []
-        retval += ["text_spk{}".format(n) for n in range(2, MAX_REFERENCE_NUM + 1)]
+
```

Comment:
Is it OK?
I just want to make sure.
There is no loop and it looks OK though

Reply:
yeah, originally I used this function to pass the auxiliary data, but then I realized that was redundant. It also caused some issues in the preprocessing, since the multispeaker tags would be passed as auxiliary data, when there was not supposed to be any aux. data.